### ISSUE TITLE
v2.2.4 spams to log with NSLog on iOs

### ISSUE THREAD

**Issue by RSATom** [rollbar/rollbar-apple#166]
After update to the latest version I see huge amount of annoying logs produced by following line:

https://github.com/rollbar/rollbar-apple/blob/9789a2d161f8d608bd19bdf7db4119bdf7cb2bfa/RollbarNotifier/Sources/RollbarNotifier/RollbarLogger.m#L1034

Is it intended?

**Comment by RSATom** [comment]
Also it would be great if there will be some way to disable `RollbarSdkLog` output ...

**Comment by akornich** [comment]
@RSATom , the noisiest SDK output can be suppressed by this configuration flag:

On a `RollbarConfig` instance used to configure the SDK, set `.developerOptions.suppressSdkInfoLogging = YES;`

but I did miss the output you mentioned. It will be fixed in the upcoming v2.3.0 release.

Thanks for reporting the issue.

**Comment by jason-gabriele-ih** [comment]
I'm still seeing a ton of logs in the console that seem unnecessary. I'm using the suppressSdkInfoLogging option.

```
[Rollbar] Column: created_at
[Rollbar] Data: 1674670633
[Rollbar] Column: destination_key
[Rollbar] Data: 1
[Rollbar] Columns: 3
```
